Sources/KlaviyoCore/AppContextInfo.swift

```diff
@@ -7,18 +7,25 @@
 import Foundation
 import UIKit
 
-public struct AppContextInfo {
-    private static let info = Bundle.main.infoDictionary
-    public static let defaultExecutable: String = (info?["CFBundleExecutable"] as? String) ??
+private let _defaultAppContextInfo: AppContextInfo? = nil
+
+@MainActor
+public func getDefaultAppContextInfo() -> AppContextInfo {
```

Comment:
Just want to ask about this - was this change something necessary with the TCA update, or was this just a simple enough change with the refactor? Seems like a decent sized change, going from a getter computed value rather than a struct with static values. I will admit though I am confused how something can be static but also computed at runtime with the ternary operator (although maybe its more compile time since we'll know about those info values then)

Reply:
Honestly I struggled with this part a lot because we had inconsistent access to this data in the sdk and some of it required main thread access. With these changes now we can at least guarantee we always access this info on the main thread.